### ISSUE TITLE
Global Python install cannot install global Python launcher

### ISSUE THREAD

**Issue by Fireforge** [lukesampson/scoop#854]
The [Python launcher](https://docs.python.org/3/using/windows.html#python-launcher-for-windows) is a very useful Windows-specific Python feature included since 3.3. During a normal Python install, this feature can either be installed in for current user only or machine-wide. When Scoop installs Python the installer is passed `InstallLauncherAllUsers=0` no matter what. So even though Scoop supposedly installs Python globally with the `--global` flag, the launcher will not be available.

This could be fixed by passing the Python installer `InstallLauncherAllUsers=1`, but only if the `--global` flag is used for the install. It seems that this kind of change in parameters based on flags is not supported, but this would be the cleanest way to implement it.

An alternative that could be implemented in the current framework is to do some `post_install` work to move the outputs of the Python launcher into the global scoop shims directory. The launcher consists of two executables, `py.exe` and `pyw.exe`, and always gets installed in this directory: `C:\Users\<username>\AppData\Local\Programs\Python\Launcher`. This can be done now, but is definitely a hack and may cause problems if the nature of the Python launcher changes in the future.

**Comment by lukesampson** [comment]
Argh. Yeah, we used to be able to handle python more cleanly and transparently when they published an MSI — I think the last nice install we had was [3.4.3](https://github.com/lukesampson/scoop/blob/6d9bca805f58b726a3a9fdf7a3d3fe06ca6a78c1/bucket/python.json).

I've just added a workaround which should fix this. You'll need to run `scoop update` and uninstall existing global installs of python to try it.
